test/DynamoCoreWpfTests/PackageManagerUITests.cs

```diff
@@ -15,6 +17,25 @@ public class PackageManagerUITests : SystemTestBase
     {
         #region Utility functions
 
+        protected void LoadPackage(string packageDirectory)
+        {
+            Model.PreferenceSettings.CustomPackageFolders.Add(packageDirectory);
+            var loader = GetPackageLoader();
+            var pkg = loader.ScanPackageDirectory(packageDirectory);
+            loader.LoadPackages(new List<Package> { pkg });
+        }
+
+        protected PackageLoader GetPackageLoader()
+        {
+            var extensions = Model.ExtensionManager.Extensions.OfType<PackageManagerExtension>();
+            if (extensions.Any())
+            {
+                return extensions.First().PackageLoader;
+            }
+
+            return null;
+        }
```

Comment:
If I remember correctly there are similar utility functions already available in other test fixtures. If so, you could try reusing them here or have them all in one place if they belong to the same class hierarchy.

Reply:
These functionss are only defined in the DynamoModelTestBase class and this PackageManagerUITests.cs inherits only the SystemTestBase. I did not notice any UI tests using these utility functions.